govuk-internal/login/template.ftl

```diff
@@ -21,7 +21,8 @@
     </#list>
 </#if>
 
-<html xmlns="http://www.w3.org/1999/xhtml" class="${properties.kcHtmlClass!}" lang="${LANG_CODE}">
+<!--[if lt IE 9]><html class="lte-ie8 ${properties.kcHtmlClass!}" lang="${LANG_CODE}"><![endif]-->
+<!--[if gt IE 8]><!--><html lang="${LANG_CODE}"><!--<![endif]-->
```

Comment:
Should the >IE8 version not be `<html class="${properties.kcHtmlClass!} lang="${LANG_CODE}">`?